src/search-param-url-generator.ts

```diff
@@ -101,6 +101,14 @@ export class SearchParamURLGenerator {
       params.append('debugging', 'true');
     }
 
+    if (searchParams.uid) {
+      params.append('uid', searchParams.uid);
+    }
+
+    if (searchParams.includeClientUrl !== false) {
```

Comment:
suggestion: use falsey so that folks don't have to send value until needed 
```suggestion
    if (searchParams.includeClientUrl) {
```

Reply:
Oh, the intent here was exactly so that no immediate caller changes are required for this param -- it's included by default *unless* the caller specifies that it shouldn't be (by passing false).

Reply:
> it's included by default unless the caller specifies that it shouldn't be (by passing false).

so truthy check should be fine, ya?  if truthy, will we need those added lines in unit tests?

Reply:
Since the type of `searchParams.includeClientUrl` is `boolean | undefined`, here's what should happen in each case:

- `false`: **don't** include the `client_url` param, by explicit request
- `true`: **do** include the `client_url` param, by explicit request (though specifying this is unnecessary)
- `undefined`: **do** include the `client_url` param, by default

Reply:
Would this be clearer if the param were `omitClientUrl`, defaulting to `false`?